roles/kubespray-defaults/defaults/main/checksums.yml

```diff
@@ -100,6 +109,8 @@ kubelet_checksums:
     v1.28.1: 0
     v1.28.0: 0
   arm64:
+    v1.31.1: fbd98311e96b9dcdd73d1688760d410cc70aefce26272ff2f20eef51a7c0d1da
+    v1.31.0: b310da449a9d2f8b928cab5ca12a6772617ba421023894e061ca2647e6d9f1c3
```

Comment:
I'm a bit unsure which one of these it's using, I can remove one if its unnecessary

Reply:
Please keep it instead of deleting it.